src/resources/js/validation.js

```diff
@@ -14,7 +16,7 @@ tribe.validation = {};
 	/**
 	 * Object containing all the selectors for Validation
 	 *
-	 * @since  4.7
+	 * @since  4.71
```

Comment:
4.7.1? or 4.71?

Reply:
@bordoni I think this was meant to be 4.7.1?

```suggestion
	 * @since  4.7.1
```